packages/block-library/src/image/edit.js

```diff
@@ -601,48 +602,44 @@ export class ImageEdit extends Component {
 					value={ align }
 					onChange={ this.updateAlignment }
 				/>
+				{ url && <MediaReplaceFlow
+					mediaURL={ url }
+					allowedTypes={ ALLOWED_MEDIA_TYPES }
+					onSelect={ this.onSelectImage }
+					onSelectURL={ this.onSelectURL }
+					onError={ this.onUploadError }
+				/> }
 				{ url && (
-					<>
-						<Toolbar>
-							<IconButton
-								className={ classnames( 'components-icon-button components-toolbar__control', { 'is-active': this.state.isEditing } ) }
-								label={ __( 'Edit image' ) }
-								aria-pressed={ this.state.isEditing }
-								onClick={ this.toggleIsEditing }
```

Comment:
We are still defining the toggleIsEditing and the is editing state although now it is not required anymore. I guess we can delete the function and all its use-cases.